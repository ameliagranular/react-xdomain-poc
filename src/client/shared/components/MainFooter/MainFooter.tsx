/**
 * The main footer of the application.
 */

import * as React from 'react';

const MainFooter: React.StatelessComponent = () => <div/>;

export { MainFooter };
