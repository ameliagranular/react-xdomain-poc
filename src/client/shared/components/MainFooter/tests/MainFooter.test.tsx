import 'jest'; // necessary for types

import { configure, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';

import { MainFooter } from '../MainFooter';

// Configure Enzyme for use with Jest & ES6+
configure({ adapter: new Adapter() });


const wrapper = (props = {}) => (
    <MainFooter {...props} />
);

const doMount = (props = {}) => {
    return mount(wrapper(props));
};

describe('MainFooter', () => {
    it('renders expected markup', () => {
        expect(toJson(doMount())).toMatchSnapshot();
    });
});
