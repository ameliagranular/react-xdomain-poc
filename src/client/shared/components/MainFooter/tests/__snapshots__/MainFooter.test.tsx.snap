// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooter renders expected markup 1`] = `
<MainFooter>
  <div />
</MainFooter>
`;
